Parsers/ASimDns/ProductParsers/VimDnsGcp.yaml

```diff
@@ -56,7 +56,7 @@ ParserQuery: |
     | where resource_type_s == "dns_query"
     //  Pre-parsing filtering:
     | where
-      (eventtype=='lookup') // -- for now we support only lookup events
+    (eventtype in ('lookup', 'Query')) // -- for now we support only lookup events
```

Comment:
Applies to all vim files: I noticed you did not change the default value.  Do you find it risky?

Reply:
Thanks!

I update for consistent handling:
- Default is always Query
- lookup is accepted as a parameter and handled as Query.